src/Makefile.am

```diff
@@ -138,3 +135,7 @@ if WITH_TCB
 		chmod $(sgidperms) $(DESTDIR)$(ubindir)/$$i; \
 	done
 endif
+if ENABLE_SUBIDS
```

Comment:
Make sure that this is opt-in. So that the distros can decide whether they want these to be `suid` binaries or `fscaps` binaries. This is important when wanting to build `shadow` in user namespaces.

Reply:
I've added a `--with-fcaps` switch to configure